src/parser.rs

```diff
@@ -44,68 +51,65 @@ pub fn get_statistic_key() -> Vec<u8> {
-            let val = key[proof_prefix.len()..].to_vec();
-            let proof = String::from_utf8(val).expect("Failed parse proof");
-            proofs.push(proof);
-            continue;
-        }
-        // Get accounts
-        if key.len() > account_prefix.len() && &key[..account_prefix.len()] == account_prefix {
-            let val = key[account_prefix.len()..].to_vec();
-            let account =
-                AccountId::try_from(String::from_utf8(val).expect("Failed parse account"))
-                    .expect("Failed parse account");
-            let account_balance = NEP141Wei::try_from_slice(
-                &base64::decode(&value.value).expect("Failed get account balance")[..],
-            )
-            .expect("Failed parse account balance");
-            accounts.insert(account, account_balance);
-            continue;
-        }
-        // Account statistics
-        if key == account_counter_key {
-            let val = base64::decode(&value.value).expect("Failed get account counter");
-            accounts_counter = read_u64(&val[..]);
-            continue;
-        }
-        // Get contract data
-        if key == get_contract_key() {
-            let val = &base64::decode(&value.value).expect("Failed get contract data")[..];
-            contract_data = FungibleToken::try_from_slice(val).expect("Failed parse contract data");
-            continue;
+        match key_type(&key) {
+            KeyType::Proof(value) => {
+                let proof = String::from_utf8(value)
+                    .map_err(|e| anyhow::anyhow!("Failed parse proof, {e}"))?;
+                proofs.push(proof);
+            }
+            KeyType::Accounts(value) => {
+                let account = AccountId::try_from(value.as_slice())
+                    .map_err(|e| anyhow::anyhow!("Failed parse account, {e}"))?;
+                let account_balance = NEP141Wei::try_from_slice(
+                    &base64::decode(&result_value.value)
+                        .map_err(|e| anyhow::anyhow!("Failed get account balance, {e}"))?,
+                )
+                .map_err(|e| anyhow::anyhow!("Failed parse account balance, {e}"))?;
+                accounts.insert(account, account_balance);
+            }
+            KeyType::Contract => {
+                let val = base64::decode(&result_value.value)
+                    .map_err(|e| anyhow::anyhow!("Failed get contract data, {e}"))?;
+                contract_data = FungibleToken::try_from_slice(&val)
+                    .map_err(|e| anyhow::anyhow!("Failed parse contract data, {e}"))?;
+            }
+            KeyType::Statistic => {
+                let val = base64::decode(&result_value.value)
+                    .map_err(|e| anyhow::anyhow!("Failed get account counter, {e}"))?;
+                accounts_counter = read_u64(&val);
+            }
+            KeyType::Unknown => anyhow::bail!("Unknown key type"),
         }
     }
     println!("Proofs: {:?}", proofs.len());
```

Comment:
Here could be used `{}` because `usize` is implemented `Display` trait. 
```suggestion
    println!("Proofs count: {:?}", proofs.len());
```